frontend/pages/review.tsx

```diff
@@ -35,29 +37,88 @@ function ErrorWrapper({ message }: { message: string | undefined }) {
 }
 
 function ApprovalList({ category, filter }: { category: ApprovalCategory; filter: ApprovalFilterType }) {
+  const { approvals, isApprovalsLoading, isApprovalsError, mutateApprovals } = useListApprovals(category, filter)
+
+  const error = MultipleErrorWrapper(
+    `Unable to load approval page`,
+    {
+      isApprovalsError,
+    },
+    ErrorWrapper
+  )
+  if (error) return error
```

Comment:
It's generally safer to put all of your return blocks for a component at the bottom. This is because:

1. It's easy to look at the bottom to see all the possible things that can be returned by the component, rather than having to read every line of code.
2. As React hooks have to run in the same order on every render, in other words they can't be used in conditional logic, having early-exit code like this can lead to someone accidentally adding a hook below it. This can easily slip by in PRs with our current setup. This would be caught by our eslint rules, but we don't currently enforce that those rules are met before being able to merge PRs.

In short, I suggest moving `if (error) return error` to just above where you're checking `isApprovalsLoading` below.